Sources/AloeStackView/Views/StackViewCell.swift

```diff
@@ -206,3 +220,16 @@ open class StackViewCell: UIView {
   }
 
 }
+
+
+extension StackViewCell: UIGestureRecognizerDelegate {
+  public func gestureRecognizer(_ gestureRecognizer: UIGestureRecognizer, shouldReceive touch: UITouch) -> Bool {
+    guard let view = gestureRecognizer.view else { return false }
+
+    let location = touch.location(in: view)
+    let hitView = view.hitTest(location, with: nil)
+
+    // ensure UIControls get the touches instead of the tap gesture
```

Comment:
Nit: To keep this comment style consistent with the other comments in the codebase, change this to
`Ensure UIControls get the touches instead of the tap gesture.`